forui/lib/src/widgets/avatar.dart

```diff
@@ -175,6 +144,83 @@ final class FAvatarStyle with Diagnosticable {
   int get hashCode => backgroundColor.hashCode ^ fadeInDuration.hashCode ^ text.hashCode;
 }
 
+class _Avatar extends StatelessWidget {
+  final FAvatarStyle? style;
+
+  /// The circle's size.
+  final double size;
+
+  /// The profile image displayed within the circle.
+  ///
+  /// If the user's initials are used, use [placeholderBuilder] instead.
+  final ImageProvider image;
+
+  /// The fallback widget displayed if [image] fails to load.
+  ///
+  /// Typically used to display the user's initials using a [Text] widget
+  /// styled with [FAvatarStyle.backgroundColor].
+  ///
+  /// Use [image] to display an image; use [placeholderBuilder] for initials.
+  final Widget Function(BuildContext)? placeholderBuilder;
+
+  const _Avatar({
+    required this.style,
+    required this.size,
+    required this.image,
+    required this.placeholderBuilder,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final style = this.style ?? context.theme.avatarStyle;
+    return Center(
```

Comment:
Do we need a `Center` widget? I think placement should be the responsibility of the user.

Reply:
This is required, it centers the widget within the circle container